ComSemApp/templates/ComSemApp/teacher/expression_form.html

```diff
@@ -97,11 +116,11 @@ <h4 class="text-muted m-b-10 font-14">
         $(function(){
             $('#recordingslist').html("<audio controls id='audioElement' src='" + "{{ object.audio.url }}" + "' type='audio/mpeg'></audio>");
             $('#deleteRecordingButton').show();
-        })
+        });
```

Comment:
love the old code issues in this code base

Reply:
<3 xD